src/components/table.js

```diff
@@ -262,6 +262,18 @@ const NoContentRow = ({ noContentMessage, noContentRenderer = _.noop, numColumns
   ])
 ])
 
+export const InputsButtonRow = ({ showRow = true, optionalButtonProps: { includeOptionalInputs, setIncludeOptionalInputs }, ...props }) => {
+  return showRow && h(div, { ...props }, [
+    h(Link,
+      {
+        style: { marginRight: 'auto' },
+        onClick: () => setIncludeOptionalInputs(includeOptionalInputs => !includeOptionalInputs)
+      },
+      [includeOptionalInputs ? 'Hide optional inputs' : 'Show optional inputs']
```

Comment:
TOL: I wonder if it would be better to right align this link? I guess I am thinking that since it is related to Attributes column, I mentally picture this on the right side of table where the Attributes column is. But I don't have a strong preference.

Reply:
<img width="1365" alt="Screenshot 2023-06-06 at 10 32 05 AM" src="https://github.com/DataBiosphere/cbas-ui/assets/67511512/451ec478-9702-4bfe-86da-0ded13265f39">

I took the left-align style from Terra UI, screenshot above. Some of the other tickets either currently on my plate or upcoming will involve adding new things to that top bar ([1962](https://broadworkbench.atlassian.net/browse/WM-1962), [1949](https://broadworkbench.atlassian.net/browse/WM-1949)), so I'm definitely open to having a conversation at some point regarding the layout of the top bar.